vulnerabilities/importers/npm.py

```diff
@@ -121,7 +121,9 @@ def _versions_to_purls(package_name, versions):
 
 
 def categorize_versions(
-    all_versions: Set[str], aff_version_range: str, fixed_version_range: str,
+    all_versions: Set[str],
+    aff_version_range: str,
```

Comment:
aff_version_range is not super obvious as a name

Reply:
Not my code, but  I see no problem in the naming.  What could be a better name ? `affected_version_range` ?

Reply:
Done